Mathlib/Data/Nat/Defs.lean

```diff
@@ -644,7 +644,6 @@ lemma one_lt_pow' (n m : ℕ) : 1 < (m + 2) ^ (n + 1) :=
  | 0 => by simp [Nat.zero_pow (Nat.pos_of_ne_zero hn)]
  | 1 => by simp
  | a + 2 => by simp [one_lt_pow hn]
-  -- one_lt_pow_iff_of_nonneg (zero_le _) h
```

Comment:
Can we find the source of this? It looks like a mistakenly left in comment, but let's make sure it wasn't something meaningful when added